packages/react/src/components/Slider/Slider-test.js

```diff
@@ -1,441 +1,480 @@
+      type(theSlider, '{arrowright}');
+      const theInput = screen.getByRole('spinbutton');
+      type(theInput, '{selectall}3');
+      expect(onChange).toHaveBeenCalledTimes(0);
+    });
+
+    describe('Error handling, expected behavior from event handlers', () => {
+      it('handles non-number typed into input field', () => {
+        const { type, tab } = userEvent;
+        renderSlider({
+          ariaLabelInput: inputAriaValue,
+          value: initialValue,
+          max: 100,
+          onChange,
+        });
+        const inputElement = screen.getByLabelText(inputAriaValue);
+        tab(); // Brings focus to slider
+        tab(); // Brings focus to input
+        type(inputElement, '{space}');
+        tab(); // Brings focus out of input
+        expect(onChange).not.toHaveBeenCalled();
+      });
+      it('gracefully tolerates empty event passed to _onDrag', () => {
+        const { mouseDown, mouseUp, mouseMove } = fireEvent;
+        const { container } = renderSlider({
+          ariaLabelInput: inputAriaValue,
+          value: 1,
+          max: 100,
+          onChange,
+        });
+        const theSlider = screen.getByRole('slider');
+        mouseDown(theSlider, { clientX: 0 });
+        mouseMove(container.firstChild, { clientX: 0 });
+        mouseUp(theSlider);
+        expect(onChange).not.toHaveBeenCalled();
+      });
+      it('gracefully tolerates empty event passed to onChange', () => {
+        const { type, tab } = userEvent;
+        renderSlider({
+          ariaLabelInput: inputAriaValue,
+          value: initialValue,
+          max: 100,
+          onChange,
+        });
+        const inputElement = screen.getByLabelText(inputAriaValue);
+        tab(); // Brings focus to slider
+        tab(); // Brings focus to input
+        type(inputElement, '{space}');
+        tab(); // Brings focus out of input
+        expect(onChange).not.toHaveBeenCalled();
+      });
+      it('should call onBlur as expected', () => {
+        const { type, tab } = userEvent;
+        renderSlider({
+          ariaLabelInput: inputAriaValue,
+          value: 10,
+          max: 100,
+          onBlur,
+        });
+        const inputElement = screen.getByLabelText(inputAriaValue);
+        tab(); // Brings focus to slider
+        tab(); // Brings focus to input
+        type(inputElement, '{space}');
+        tab(); // Brings focus out of input
+        expect(onBlur).toHaveBeenCalledTimes(2);
+      });
+      it('should call onKeyDown as expected', () => {
+        const { type, click } = userEvent;
+        renderSlider({
+          ariaLabelInput: inputAriaValue,
+          value: initialValue,
+          max: 100,
+          onChange,
+        });
+        const theSlider = screen.getByRole('slider');
+        click(theSlider);
+        type(theSlider, '{arrowright}');
+        type(theSlider, '{arrowright}');
+        expect(onChange).toHaveBeenCalledTimes(3);
```

Comment:
@matthewgallo It looks like this one is failing in CI, `onChange` is being called 4 times.

Reply:
Hey @tay1orjones, was able to get tests all passing in CI now, but Percy seems to still be failing. Any suggestions there?